src/java/main/org/apache/zookeeper/server/quorum/QuorumPeer.java

```diff
@@ -682,27 +682,19 @@ public void setQuorumAddress(InetSocketAddress addr){
     }
 
     public InetSocketAddress getElectionAddress(){
-        synchronized (QV_LOCK) {
-            return myElectionAddr;
-        }
+        return myElectionAddr;
```

Comment:
Here is why. 
* All set code path was protected by QV_LOCK already, which implies that whoever calls set* should already acquire the QV_LOCK.
* On get code path, it is not guaranteed that caller will hold QV_LOCK while a reconfig op is in flight, and I think it is possible in theory we could get out dated election address, e.g. at [this place](https://github.com/apache/zookeeper/blob/master/src/java/main/org/apache/zookeeper/server/quorum/QuorumCnxManager.java#L265). Hopefully this is not a problem because we don't require precise information being sent here - if we get out dated addr (in case the current quorum peer is being reconfigured) and sent this to another peer, another peer will not able to connect but that's fine, it will retry until at certain point later it will get correct information.